src/Listeners/SaveUserBio.php

```diff
@@ -45,7 +52,7 @@ public function handle(Saving $event)
         if (isset($attributes['bio'])) {
             $actor->assertCan('editBio', $user);
 
-            $user->bio = Str::limit($attributes['bio'], $this->settings->get('fof-user-bio.maxLength'), '');
+            $user->bio = $this->formatter->parse(Str::limit($attributes['bio'], $this->settings->get('fof-user-bio.maxLength'), ''));
```

Comment:
Would it be better to use a validator to check the length, to make error message better? I can't think of any extension that breaks at the parser level from an invalid content syntax, but by validating the length first we are sure we don't cause an error to be thrown because the input to the parser contains half a bbcode or markdown tag.

Reply:
Not sure if the way I did it is the best way for retrieving the max bio length from DB.

https://github.com/FriendsOfFlarum/user-bio/blob/e979a1f07989a911aa6dce3fe415523624021d43/src/Validators/UserBioValidator.php#L32